metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -37,6 +37,7 @@ case class UserConfiguration(
     superMethodLensesEnabled: Boolean = false,
     showInferredType: Boolean = false,
     showImplicitArguments: Boolean = false,
+    showImplicitConversions: Boolean = false,
```

Comment:
Actually this also show implicit class... so not sure about the name. What do you all think?

Reply:
Since this is just a setting name I'm not that opposed to it being super explicit. I don't love it, but `showImplicitConversionsAndClasses` 🤔

Reply:
Changed! I think it makes sense to be super explicit here.